featurebyte/feast/utils/registry_construction.py

```diff
@@ -925,7 +926,6 @@ def create_feast_registry_proto_for_feature_materialization(
             feature_services=[],
             on_demand_feature_views=[],
             stream_feature_views=[],
-            request_feature_views=[],
         )
```

Comment:
Remove this attribute as it is no longer valid in the newer `RepoContents` (from `feast` package).